components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml

```diff
@@ -347,6 +357,7 @@
 <config_land_ice_flux_mode ocn_grid="SOwISC12to60E2r4">'pressure_only'</config_land_ice_flux_mode>
 <config_land_ice_flux_mode ocn_grid="ECwISC30to60E2r1">'pressure_only'</config_land_ice_flux_mode>
 <config_land_ice_flux_mode ocn_grid="IcoswISC30E3r5">'pressure_only'</config_land_ice_flux_mode>
+<config_land_ice_flux_mode ocn_grid="IcosXISC30E3r7">'pressure_only'</config_land_ice_flux_mode>
```

Comment:
Just noting that, even though ice-shelf cavities have been culled, `landIcePressure` is not zero in parts of the open ocean so this is needed.